beacon-chain/blockchain/chain_info.go

```diff
@@ -73,14 +72,19 @@ type CanonicalFetcher interface {
 }
 
 // FinalizationFetcher defines a common interface for methods in blockchain service which
-// directly retrieves finalization and justification related data.
+// directly retrieve finalization and justification related data.
 type FinalizationFetcher interface {
 	FinalizedCheckpt() *ethpb.Checkpoint
 	CurrentJustifiedCheckpt() *ethpb.Checkpoint
 	PreviousJustifiedCheckpt() *ethpb.Checkpoint
 	VerifyFinalizedBlkDescendant(ctx context.Context, blockRoot [32]byte) error
 }
 
+// OptimisticSyncFetcher retrieves information about optimistic sync.
+type OptimisticSyncFetcher interface {
```

Comment:
```suggestion
type OptimisticFetcher interface {
```

or `OptimisticModeFetcher`

Reply:
`OptimismFetcher` :smile: